-- run_test.py: invoking following command in '/workspaces/isaac_ros-dev/isaac_ros_nitros/isaac_ros_nitros_type/isaac_ros_nitros_std_msg_type':
 - /opt/ros/humble/bin/ament_cpplint --xunit-file /workspaces/isaac_ros-dev/build/isaac_ros_nitros_std_msg_type/test_results/isaac_ros_nitros_std_msg_type/cpplint.xunit.xml

Using '--root=/workspaces/isaac_ros-dev/isaac_ros_nitros/isaac_ros_nitros_type/isaac_ros_nitros_std_msg_type/include' argument

Done processing /workspaces/isaac_ros-dev/isaac_ros_nitros/isaac_ros_nitros_type/isaac_ros_nitros_std_msg_type/include/isaac_ros_nitros_std_msg_type/nitros_int64.hpp

Using '--root=/workspaces/isaac_ros-dev/isaac_ros_nitros/isaac_ros_nitros_type/isaac_ros_nitros_std_msg_type/src' argument

Done processing /workspaces/isaac_ros-dev/isaac_ros_nitros/isaac_ros_nitros_type/isaac_ros_nitros_std_msg_type/src/nitros_int64.cpp

Using '--root=/workspaces/isaac_ros-dev/isaac_ros_nitros/isaac_ros_nitros_type/isaac_ros_nitros_std_msg_type/test/src' argument

Done processing /workspaces/isaac_ros-dev/isaac_ros_nitros/isaac_ros_nitros_type/isaac_ros_nitros_std_msg_type/test/src/nitros_int64_forward_node.cpp

No problems found


-- run_test.py: return code 0
-- run_test.py: verify result file '/workspaces/isaac_ros-dev/build/isaac_ros_nitros_std_msg_type/test_results/isaac_ros_nitros_std_msg_type/cpplint.xunit.xml'
